cypress/integration/pages/testsForAllAMPPages.js

```diff
@@ -12,7 +13,7 @@ export const testsThatFollowSmokeTestConfigForAllAMPPages = ({
   pageType,
 }) => {
   describe(`Running testsForAllAMPPages for ${service} ${pageType}`, () => {
-    if (pageType !== 'errorPage404') {
+    if (!pageTypeSomeTestsNotRun) {
```

Comment:
Why are we not running some tests for the most read page?

Reply:
I think we can run testForAllAMPPages and testsForAllCanonicalPages for most read, but in this test `cypress/integration/pages/testsForAllPages.js` there is a test that checks the JSON response of the page for metadata (most read doesn't have this)

```
          cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
```

I'll update the test so that it is not run specifically for most read page here ^

Reply:
even error pages don't have metadata.